components/data/data-management/src/main/java/org/eclipse/dirigible/components/data/management/format/ResultSetCsvWriter.java

```diff
@@ -152,7 +152,7 @@ public void write(ResultSet resultSet, OutputStream output) throws Exception {
                         } else if (dbt == Types.OTHER) {
                             Object dataObject = resultSet.getObject(name);
                             if (dataObject instanceof PGobject) {
-                                if (value == null && stringify) {
+                                if (stringify) {
```

Comment:
This seems strange, why is it needed to set `value = "[NULL]"` if it's re-set on the next line? Is there a missing `else` branch?

Reply:
fixed